src/components/ChattyLLM/ChattyLLMInputForm.vue

```diff
@@ -219,20 +219,69 @@ export default {
 
 	watch: {
 		async active() {
+			this.loading.llmGeneration = false
+			this.loading.titleGeneration = false
```

Comment:
let's set them to true. For llm generation when we do a network request to check if there is a running task, it switches from false to true quickly resulting in a glitchy effect. Since we're going to do this check on every session change, let's first assume there is a task and block the input. Later we free the input when there are no tasks.